Neos.EventSourcedContentRepository/Classes/Domain/Context/NodeAggregate/Property/Normalizers/DoctrinePersistentObjectNormalizer.php

```diff
@@ -37,6 +38,7 @@ public function normalize($object, string $format = null, array $context = [])
     public function supportsNormalization($data, string $format = null)
     {
         return (
+            is_subclass_of(TypeHandling::getTypeForValue($data), ResourceBasedInterface::class, true) ||
```

Comment:
just a temporary/unrelated hack to make this work with latest master

Reply:
I think I solved this with https://github.com/neos/contentrepository-development-collection/pull/137 :)